content/docs/dvclive/live/log_artifact.md

```diff
@@ -26,6 +39,10 @@ Uses `dvc add` to track `path` with DVC, generating a `{path}.dvc` file.
 When combined with `save_dvc_exp=True`, it will ensure that `{path}.dvc` is
 included in the experiment.
 
+If `Live` was initialized with `dvcyaml=True`, will add an artifact to
+corresponding `dvc.yaml`, marking it as a `model` for DVC and making artifact
+appear in [Studio Model Registry](/doc/studio).
```

Comment:
I think we need to mention that it will save any metadata provided, but the model registry will only look for `type=model` artifacts. Let's also mention that this is a WIP and the model registry will not do anything right now.

Reply:
Another thing to mention: if a name is not provided, the path stem (last part of the path without the file extension) will be used as the artifact name.